.env.production

```
@@ -0,0 +1,37 @@
+# DevSkyy Production Environment Configuration
+# Generated: 2025-11-16
+# IMPORTANT: Keep this file secure and never commit to version control
+
+# Security
+SECRET_KEY=EH1FQvHj_7w_qncjj-CHfaQfbut9OlMNs1d4SnGMNWg
+JWT_SECRET_KEY=EH1FQvHj_7w_qncjj-CHfaQfbut9OlMNs1d4SnGMNWg
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_SECRET_KEY key should go before the SECRET_KEY key

(UnorderedKey)